test/libsolidity/ASTJSON/fail_after_parsing.sol

```diff
@@ -1,12 +1,9 @@
 function g() public;
-
 interface I {
     struct S { S s; }
-
     function f(E storage e) {
         error E;
         emit E();
-
         ++c;
         uint calldata c = 123.4;
     }
```

Comment:
There are no actual changes here, so I'd exclude this file completely from the PR.